### PR TITLE
acrn.conf: remove maxcpus from cmdline

### PR DESCRIPTION
**Pull request by JasonChenCJ**
after SOS start with all CPUs, it should not set maxcpus in cmdline

Signed-off-by: Jason Chen CJ <jason.cj.chen@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

File: hypervisor/bsp/uefi/clearlinux/acrn.conf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 title The ACRN Service OS
 linux   /EFI/org.clearlinux/kernel-org.clearlinux.pk414-sos.4.14.52-63
-options pci_devices_ignore=(0:18:1) maxcpus=1 console=tty0 console=ttyS0 i915.nuclear_pageflip=1 root=PARTUUID=<UUID of rootfs partition> rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.tsd_init=7 i915.tsd_delay=2000 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2 i915.enable_gvt=1 i915.enable_initial_modeset=1 hvlog=2M@0x1FE00000
```
Comment (dongyaozu) [inline hypervisor/bsp/uefi/clearlinux/acrn.conf]: do you need to change the script when starting UOS?

**Comment by lijinxia** [comment]
OK to verify

File: hypervisor/bsp/uefi/clearlinux/acrn.conf (hunk 1 above)
Comment (lijinxia) [reply hypervisor/bsp/uefi/clearlinux/acrn.conf]: The change for launch UOS script has been merged in #572

**Pull request merged**